dxfeed/adapter.js

```diff
@@ -0,0 +1,52 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+const username = process.env.API_USERNAME
+const password = process.env.API_PASSWORD
+const DEMO_ENDPOINT = 'https://tools.dxfeed.com/webservice/rest'
```

Comment:
Any special reason why you named it `DEMO_ENDPOINT`?

```suggestion
const DEFAULT_ENDPOINT = 'https://tools.dxfeed.com/webservice/rest'
```

Reply:
Because it's actually just a demo endpoint, and I don't think we should trust any data that comes from that endpoint as far as I understand.

The node operators should be provided an endpoint along with credentials when they get set up with an account with dxFeed.